libs/application/templates/university/src/fields/ModeOfDeliverySelection/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { FieldBaseProps } from '@island.is/application/types'
+import { FC, useState } from 'react'
+import { Box } from '@island.is/island-ui/core'
+import { UniversityApplication } from '../../lib/dataSchema'
+import { Routes } from '../../lib/constants'
+import {
+  RadioController,
+  SelectController,
+} from '@island.is/shared/form-fields'
+import { ProgramBase } from '@island.is/clients/university-gateway-api'
+import { information } from '../../lib/messages'
+import { useLocale } from '@island.is/localization'
+import { getValueViaPath } from '@island.is/application/core'
+
+export const ModeOfDeliverySelection: FC<FieldBaseProps> = ({
```

Comment:
Why cant this be solved using a buildSelectField with a condition and an options function that has access to the application object?

Reply:
yes fixed